dashboard/test/models/course_version_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CourseVersionTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
the manual testing you've done in rails console would be easy to add as a unit test here.